modular_skyrat/modules/command_vendor/code/vending/vending.dm

```diff
@@ -10,39 +10,47 @@
 
 /obj/machinery/vending/access/command/build_access_list(list/access_lists)
 	access_lists["[ACCESS_CAPTAIN]"] = list(
+		// CAPTAIN
 		/obj/item/clothing/head/caphat = 1, /obj/item/clothing/head/caphat/beret = 1, /obj/item/clothing/head/caphat/beret/alt = 1,
 		/obj/item/clothing/under/rank/captain = 1, /obj/item/clothing/under/rank/captain/skirt = 1, , /obj/item/clothing/under/rank/captain/humble = 1,
 		/obj/item/clothing/under/rank/captain/dress = 1, /obj/item/clothing/head/caphat/parade = 1,
 		/obj/item/clothing/under/rank/captain/parade = 1, /obj/item/clothing/suit/toggle/captains_parade = 1,
-		/obj/item/clothing/suit/captunic = 1, /obj/item/storage/backpack/captain = 1, /obj/item/storage/backpack/satchel/cap = 1,
-		/obj/item/storage/backpack/duffelbag/captain = 1, /obj/item/clothing/shoes/sneakers/brown = 1
+		/obj/item/clothing/suit/captunic = 1, /obj/item/clothing/neck/mantle/capmantle = 1, /obj/item/storage/backpack/captain = 1,
+		/obj/item/storage/backpack/satchel/cap = 1, /obj/item/storage/backpack/duffelbag/captain = 1, /obj/item/clothing/shoes/sneakers/brown = 1,
+
+		// BLUESHIELD
+		/obj/item/clothing/head/beret/blueshield = 1, /obj/item/clothing/head/beret/blueshield/navy = 1,
+		/obj/item/clothing/under/rank/security/blueshield = 1, /obj/item/clothing/under/rank/security/blueshieldturtleneck = 1,
+		/obj/item/clothing/suit/armor/vest/blueshield = 1, /obj/item/clothing/neck/mantle/bsmantle = 1,
+		/obj/item/storage/backpack/blueshield = 1, /obj/item/storage/backpack/satchel/blueshield = 1,
+		/obj/item/storage/backpack/duffel/blueshield = 1, /obj/item/clothing/shoes/laceup = 1
```

Comment:
If you want to split things on multiple lines, let's do it right all the way.
Put only one element per line here, I'd do a suggestion but GitHub isn't happy about letting us make suggestions on something that has removed lines, for some reason. This is something that should be done all throughout the file, that being said.